mne/dipole.py

```diff
@@ -346,6 +346,14 @@ class DipoleFixed(object):
     verbose : bool, str, int, or None
         If not None, override default verbose level (see :func:`mne.verbose`
         and :ref:`Logging documentation <tut_logging>` for more).
+    pos : array, shape (n_dipoles, 3)
+        The dipoles positions (m) in head coordinates.
+    amplitude : array, shape (n_dipoles,)
+        The amplitude of the dipoles (Am).
+    ori : array, shape (n_dipoles, 3)
+        The dipole orientations (normalized to unit length).
+    gof : array, shape (n_dipoles,)
+        The goodness of fit.
```

Comment:
Ok! I'll get these attributes from fit_dipole(..) in test_rap_music.py. I'll just check if I can get the same values as the originals. And try to show this in another commit. Thank you